templates/main/group_overview.html.ep

```diff
@@ -39,16 +39,18 @@
 %= include 'main/group_builds', build_results => $build_results, group => $group, children => undef, default_expanded => 1
 %= include 'main/more_builds', limit_builds => $limit_builds
 
-<h2>Comments</h2>
-% for my $comment (reverse @$comments) {
-    %= include 'comments/comment_row', comment_id => $comment->id, comment => $comment, user => $comment->user, context => {type => 'group', id => $group->{id}}, put_action => 'apiv1_put_group_comment', delete_action => 'apiv1_delete_group_comment'
-% }
+% if ($comments_enabled) {
```

Comment:
Couldn't this just be `% if (@$comments) {`? Then only the controller code has to deal with comment limits.